lib/stripe_mock/request_handlers/issuing_tokens.rb

```diff
@@ -0,0 +1,13 @@
+module StripeMock
+  module RequestHandlers
+    module IssuingTokens
+      def IssuingTokens.included(klass)
+        klass.add_handler 'get /v1/issuing/tokens', :list_issuing_tokens
+      end
+
+      def list_issuing_tokens(route, method_url, params, headers)
+        Data.mock_list_object(issuing_tokens, params)
```

Comment:
In theory this mock should apply the filtering that is available through `params`, but I don't really need that right now.